Resources/Prototypes/Research/technology.yml

```diff
@@ -0,0 +1,621 @@
+  discipline: Industrial
+  tier: 2
+  cost: 5000
+  recipeUnlocks:
+  - HolofanProjector
+  - PortableScrubberMachineCircuitBoard
+
+- type: technology
+  id: SuperPowercells
+  name: research-technology-super-powercells
+  icon:
+    sprite: Objects/Power/power_cells.rsi
+    state: high
+  discipline: Industrial
+  tier: 2
+  cost: 7500
+  recipeUnlocks:
+  - PowerCellHigh
+  technologyPrerequisites:
+  - AdvancedPowercells
+
+# Tier 3
+
+- type: technology
+  id: BluespaceStorage
+  name: research-technology-bluespace-storage
+  icon:
+    sprite: Clothing/Back/Backpacks/holding.rsi
+    state: holding
+  discipline: Industrial
+  tier: 3
+  cost: 10000
+  recipeUnlocks:
+  - ClothingBackpackHolding
+  - ClothingBackpackSatchelHolding
+  - ClothingBackpackDuffelHolding
+
+# Biochemical
+# Tier 1
+
+- type: technology
+  id: Chemistry
+  name: research-technology-chemistry
+  icon:
+    sprite: Objects/Specific/Chemistry/beaker_large.rsi
+    state: beakerlarge
+  discipline: Biochemical
+  tier: 1
+  cost: 10000
+  recipeUnlocks:
+  - LargeBeaker
+  - Dropper
+  - HotplateMachineCircuitboard
+  - ChemicalPayload
+
+- type: technology
+  id: SurgicalTools
+  name: research-technology-surgical-tools
+  icon:
+    sprite: Objects/Specific/Medical/Surgery/saw.rsi
+    state: saw
+  discipline: Biochemical
+  tier: 1
+  cost: 5000
+  recipeUnlocks:
+  - Scalpel
+  - Retractor
+  - Cautery
+  - Drill
+  - Saw
+  - Hemostat
+
+- type: technology
+  id: BiochemicalStasis
+  name: research-technology-biochemical-stasis
+  icon:
+    sprite: Structures/Machines/stasis_bed.rsi
+    state: icon
+  discipline: Biochemical
+  tier: 1
+  cost: 10000
+  recipeUnlocks:
+  - CryostasisBeaker
+  - StasisBedMachineCircuitboard
+
+- type: technology
+  id: Virology
+  name: research-technology-virology
+  icon:
+    sprite: Structures/Machines/diagnoser.rsi
+    state: icon
+  discipline: Biochemical
+  tier: 1
+  cost: 5000
+  recipeUnlocks:
+  - VaccinatorMachineCircuitboard
+  - DiagnoserMachineCircuitboard
+
+# Tier 2
+
+- type: technology
+  id: Cryopod
+  name: research-technology-cryogenics
+  icon:
+    sprite: Structures/Machines/cryogenics.rsi
+    state: pod-on
+  discipline: Biochemical
+  tier: 2
+  cost: 7500
+  recipeUnlocks:
+  - CryoPodMachineCircuitboard
+
+- type: technology
+  id: ChemicalDispensary
+  name: research-technology-chemical-dispensary
+  icon:
+    sprite: Structures/dispensers.rsi
+    state: industrial-working
+  discipline: Biochemical
+  tier: 2
+  cost: 10000
+  recipeUnlocks:
+  - ChemMasterMachineCircuitboard
+  - ChemDispenserMachineCircuitboard
+  technologyPrerequisites:
+  - Chemistry
+
+- type: technology
+  id: CrewMonitoring
+  name: research-technology-crew-monitoring
+  icon:
+    sprite: Objects/Specific/Medical/handheldcrewmonitor.rsi
+    state: scanner
+  discipline: Biochemical
+  tier: 2
+  cost: 10000
+  recipeUnlocks:
+  - HandheldCrewMonitor
+
+# Tier 3
+
+- type: technology
+  id: Cloning
```

Comment:
Cloning being a T3 tech seems kinda strange, since cloning is mapped in and available on roundstart. I think this will lead to a lot of early shuttle calls if cloning gets bombed and sci already unlocked a T3 from a different discipline.

Reply:
arguments against:
- cloning is rarely needed with good med staff since defibs cover 90% of cases
- cloning is not available roundstart on at least 1 map so yeah